lib/internal/streams/end-of-stream.js

```diff
@@ -7,20 +7,12 @@ const {
   ERR_INVALID_ARG_TYPE,
   ERR_STREAM_PREMATURE_CLOSE
 } = require('internal/errors').codes;
+const { once } = require('internal/util');
 
 function isRequest(stream) {
   return stream.setHeader && typeof stream.abort === 'function';
 }
 
-function once(callback) {
-  let called = false;
-  return function(err) {
-    if (called) return;
-    called = true;
-    callback.call(this, err);
```

Comment:
It seems like the callback has a different context in `once` from `internal/util`. Is this something we want to pay attention to / align the behavior?

Reply:
Yeah, thanks – I’ve made the one in `internal/util` more generic :+1: